### DISCUSSION TITLE
Flows are turned off after a solution import even though the --activate-plugins flag is set on the call

### DISCUSSION THREAD

**Discussion by jordanabakerafs** [microsoft/powerplatform-build-tools#950] [Q&A]
We're running a solution import in an ADO pipeline which calls
`pac solution import --path <path> --async true --force-overwrite true --publish-changes false --skip-dependency-check false --convert-to-managed false --max-async-wait-time 180 --activate-plugins true --skip-lower-version false --import-as-holding true --settings-file <filepath>`

Recently we've begun seeing some flows turned off after import where we need to go back in and manually turn them on.  

There doesn't appear to be any pattern to which ones turn off or why.  We were hoping the community may have some ideas for which direction to look to start troubleshooting.

Thanks for the assist!

**Comment by isaacmissinghamwt** [comment]
We are also having this issue - we had to swap back to the deprecated "Activate processes" flag, which seems to be working okay (or at least, it isn't turning everything off!)
